### ISSUE TITLE
Assess accessiblity of Icon

### ISSUE THREAD

**Issue by tobiasljungstrom** [SKF/ferris#235]
Asses the accessibility of the Icon component and mark it in the component table.

The component should meet the following criteria (if applicable):

- [x] Tab-able with a clear focus state
- [x] Screen reader should be able to traverse it
- [x] Custom control should work with screen reader
- [x] Focus should be directed to any new items that pop up on the screen
- [x] Color and contrast should be good

**Comment by tobiasljungstrom** [comment]
Almost nothing applies since icons are aria-hidden. Complementing documentation should be provided so that the user sets an accompanying (possibly hidden) descriptive text.
